iexec-core-library/src/main/java/com/iexec/core/api/SchedulerClient.java

```diff
@@ -58,7 +58,7 @@ public interface SchedulerClient {
 
     @RequestLine("POST /workers/register")
     @Headers("Authorization: {authorization}")
-    void registerWorker(@Param("authorization") String authorization, WorkerModel model);
+    Void registerWorker(@Param("authorization") String authorization, WorkerModel model);
```

Comment:
Is there a reason why we need a `Void` return type there?

Reply:
This change is due to generic use in worker